### DISCUSSION TITLE
CsvReader not applying configuration comment property when it is set as '#'

### DISCUSSION THREAD

**Discussion by kohtianan** [JoshClose/CsvHelper#1282] [Q&A]
if i set the Comment property as the default('#'), csvreader still reads commented lines as input

var config = new Configuration
            {
                Delimiter = ",",
                //Comment = '#' //this does not work although # is the default
                Comment = Convert.ToChar("#"), //this works
                Quote = '"',
                HasHeaderRecord = false,
                AllowComments = true,
            };

**Comment by JoshClose** [comment]
What do you mean by
> ...still reads commented lines as input

**Comment by doon386** [comment]
For me neither work. I have a input file that contains lines beginning with '#' for comments. 
e.g. # Added 4 SEPT 2018

The 'comment'  lines are still read in as if there was no comment there regardless if I set the comment char to # or take the default.

I'm currently using v12.1.2
